dns/checkip/src/etc/inc/plugins.inc.d/dyndns/checkipsrvsplugin.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+if (class_exists('OPNsense\DynDNS\CheckIP_Service') && 
+    class_exists('OPNsense\DynDNS\CheckIP_Factory_Default_Service')) {
+
+    // Get the check IP services.
+    $mdlCheckIPService = new OPNsense\DynDNS\CheckIP_Service();
+    $a_checkipservices = $mdlCheckIPService->service->getNodes();
+
+    // Append the factory default check IP service to the list.
+    $mdlCheckIPFDS = new OPNsense\DynDNS\CheckIP_Factory_Default_Service();
+    $a_checkipservices['FDS'] = $mdlCheckIPFDS->factory_default_service->getNodes();
+
+    // Is the factory default service disabled?
+    if ($mdlCheckIPService->disable_factory_default_service->__toString() == "1") {
+        $a_checkipservices['FDS']['default'] = 0;
+    }
+} else {
+    $ip_address = 'Check IP services not available';
+    return $ip_address;
```

Comment:
this is:

return 'Check IP services not available';

Reply:
changed